ProcessMaker/processmaker
# FOUR-13914 | The Language Selected for a User Is Not Complete

# Issue
Ticket: [FOUR-13914](https://processmaker.atlassian.net/browse/FOUR-13914)

When a language is selected for a user, all text is not translated into the selected language.

# Solution
Update the en.json file in `processmaker` to ensure the missing text translations are accounted for.

# How to Test
1. Go to branch `observation/FOUR-13914` in `processmaker`.
2. Create a user. Change their language to `es` for Spanish.
3. Ensure that the text on the following pages is translated into Spanish:
	- Homepage (Custom Dashboard text)
	- Processes
	- Tasks
	- Requests
  - Designer
	- Admin
	- +CASE modal

ci:next

# Code Review Checklist


[FOUR-13914]: https://processmaker.atlassian.net/browse/FOUR-13914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
